Test/components/HV_Iox/Test_HV_Iox.cpp

```diff
@@ -0,0 +1,69 @@
+// Author: Marco
```

Comment:
Perhaps add some comments to these tests so that someone who doesn't know what the functions are can tell what exactly is failing/passing

Reply:
👌